install_ant-media-server.sh

```diff
@@ -21,6 +21,20 @@ DEFAULT_JAVA="$(readlink -f $(which java) | rev | cut -d "/" -f3- | rev)"
 LOG_DIRECTORY="/var/log/antmedia"
 ARCH=`uname -m`
 
+init() {
+  remote_file="$(curl -sL https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh | md5sum | cut -d ' ' -f 1)"
+  local_file="$(md5sum $0 | cut -d '' -f 1 )"
+
+  if [ "$remote_file" != "$local_file" ]; then
+    wget -O $0 -q https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh
+    chmod +x $0
+    echo "Updating the installation script. Please rerun the script."
+    exit 1
+  fi
+}
+
+init
+
```

Comment:
I see a couple of issues in here. 

1. Some users have headless installation in the workflow so that having this will break their flow. 
2. Some users may not prefer to update the installation script because of some internal reasons. I think we should not force them to update the script.

Reply:
I can add a notification or input, "Do you want to update it ?". Sounds good ?